take-homes/be-take-home.md

```diff
@@ -0,0 +1,48 @@
+# Tea Subscription Service
+
+You will create a Rails API for a Tea Subscription Service.
+
+## Requirements:
+
+At a minimum, you must create:
+
+- An endpoint to subscribe a customer to a tea subscription
+- An endpoint to cancel a customer's tea subscription
+- An endpoint to see all of a customer's subsciptions (active and cancelled)
+
+Beyond the base requirements, include any additional endpoints you'd like, but be mindful of your time. You don't need to go overboard, consider what might be necessary to achieve MVP so that this API is easy to understand and consume by a Frontend Developer.
+
+## Things we'll be looking for:
+
+- A strong understanding of Rails
+- Ability to create restful routes
+- Demonstration of well-organized code, following OOP
+- Test Driven Development
+- Clear documentation
+
+Try to limit your total time to approximately 6 hours on this project. Prioritize what is important to completing MVP and demonstrating your capabilities as a developer. If you are looking for a stretch option, you can consume this API for tea information: [Tea API](https://tea-api-vic-lo.herokuapp.com/)
+
+### Data Details
+
+Note: The below data outlines are meant to be a guide for data details you would want use. If you want to change or include more information please do so. Using fake data in your application is okay.
```

Comment:
I like this as it gives students an idea on the types of data and tables there will be and pushes them to determine the types of relationships they might have.  Out of curiosity because I'm less familiar with these kind of interviews, is it common for most BE take home challenges to offer these kind of details?

Reply:
In my experience, I've had "starter" code that has included these details whether that's the classes already exist with some of this information or having tests to help guide the development. I wanted each of them to create their own repo rather than us needing to maintain a starter at least for this first time giving the project. So I wanted to be sure to share those details here in the prompt.